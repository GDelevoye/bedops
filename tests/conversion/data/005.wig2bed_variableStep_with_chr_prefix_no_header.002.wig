variableStep chrom=chr2 span=5
300701 12.5
