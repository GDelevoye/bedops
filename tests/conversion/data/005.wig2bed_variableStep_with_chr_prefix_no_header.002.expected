chr2	300700	300705	id-1	12.500000
